README.md

```diff
@@ -1,8 +1,48 @@
 [![Build Status](https://travis-ci.org/cowboyd/microstates.js.svg?branch=master)](https://travis-ci.org/cowboyd/microstates.js)
 [![Coverage Status](https://coveralls.io/repos/github/cowboyd/microstates.js/badge.svg?branch=master)](https://coveralls.io/github/cowboyd/microstates.js?branch=master)
+[![License: MIT](https://img.shields.io/badge/License-MIT-yellow.svg)](https://opensource.org/licenses/MIT)
 
 # Microstates
 
+* [Intro](#intro)
+* [microstate constructor](#microstate-constructor)
+* [Composition](#composition)
+* [Static values](#static-values)
+* [Computed Properties](#computed-properties)
+* [Transitions](#transitions)
+* [Batch Transitions](#batch-transitions)
+* [Changing structure](#changing-structure)
+  * [Parameterized Arrays & Objects](#parameterized-arrays--objects)
```

Comment:
Parameterized Data & Observable Interoperability shouldn't be nested under Changing structure.